solr/solrj/src/java/org/apache/solr/client/solrj/response/QueryResponse.java

```diff
@@ -102,21 +101,10 @@ public class QueryResponse extends SolrResponseBase {
   private Map<String, Object> _debugMap = null;
   private Map<String, Object> _explainMap = null;
 
-  // utility variable used for automatic binding -- it should not be serialized
-  private final transient SolrClient solrClient;
+  public QueryResponse() {}
 
-  public QueryResponse() {
-    solrClient = null;
-  }
-
-  /** Utility constructor to set the solrServer and namedList */
-  public QueryResponse(NamedList<Object> res, SolrClient solrClient) {
+  public QueryResponse(NamedList<Object> res) {
```

Comment:
I love this, but also, does this need to be deprecated and or flagged?  Seems like a big method chagne?

Reply:
Users shouldn't be using this constructor; it's for SolrJ internals.